tests/BotTest.php

```diff
@@ -781,3 +804,25 @@ public function handleEvent()
         // left empty for stubbing
     }
 }
+
+/**
+ * Plugin class which implements PluginContainerInterface.
+ */
+class TestPluginContainer extends AbstractPlugin implements PluginContainerInterface
```

Comment:
Problem is, Phake can't mock more than one interface, so then the object isn't a valid plugin and it won't be accepted by the bot.